container_files/tutorials/Using the jseval function inside an SQL statement.ipynb

```diff
@@ -0,0 +1,365 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using Javascript inside MLDB's Query Language statements with the built-in jeval function\n",
```

Comment:
Using Javascript inside MLDB's Query Language statements with the built-in jeval function -> Executing Javascript code directly in SQL statements using the jseval function